common.env

```diff
@@ -1,11 +1,8 @@
-SNAPP_CONTRACT_ADDRESS=0xe982E462b094850F12AF94d21D470e21bE9D0E9C
-SNAPP_BASE_LIB_ADDRESS=0x9561C133DD8580860B6b7E504bC5Aa500f0f06a7
-STABLEX_CONTRACT_ADDRESS=0x67B5656d60a809915323Bf2C40A8bEF15A152e3e
```

Comment:
Where have these gone?

Reply:
They are no longer needed - its one of the bonuses of using `ethcontract`. It reads it from the truffle artifacts (`BatchExchange.json` and `SnappAuction.json`).

Reply:
This should mean that updating contract deployments should just require submodule updates (unless there are breaking changes of course).